code/modules/reagents/dispenser/dispenser2.dm

```diff
@@ -74,16 +77,18 @@
 /obj/machinery/chemical_dispenser/attackby(obj/item/W, mob/user)
 	if(istype(W, /obj/item/chems/chem_disp_cartridge))
 		add_cartridge(W, user)
+		return TRUE
 
-	else if(isScrewdriver(W))
+	if(isCrowbar(W) && length(cartridges))
```

Comment:
Should check `panel_open` to avoid interaction clashes. This makes it non-deconstructable.

Reply:
The idea here was that it would be deconstructable if there weren't any cartridges left, but because cartridges get dumped anyway, I added the panel_open check.